### PR TITLE
Prepare for nette 3.0

### PR DESCRIPTION
**Pull request by MartinSzollos2016**

### CODE REVIEW

**Comment by akadlec** [comment]
You have to fix tests

**Comment by MartinSzollos2016** [comment]
https://github.com/iPublikuj/flash-messages/pull/5/commits/ab5d5d29bc1da0a0d18d04fa61318bd84c93be59

Toto bych ještě přidal do verze 2.0.10

File: src/IPub/FlashMessages/TFlashMessages.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function injectFlashMessages(Components\IControl $flashMessagesFactory, F
 	 */
 	public function flashMessage($message, string $type = 'info', ?string $title = null, bool $overlay = false, ?int $count = null, ?array $parameters = []): \stdClass
 	{
-		$this->flashNotifier->message($message, $level, $title, $overlay, $count, $parameters);
+		return $this->flashNotifier->message($message, $type, $title, $overlay, $count, $parameters);
```
Comment (akadlec) [inline src/IPub/FlashMessages/TFlashMessages.php:64]: Nice catch!

**Review by akadlec — APPROVED** [review]

**Review by akadlec — APPROVED** [review]

**Comment by MartinSzollos2016** [comment]
Thanks for merging. Could you please update library on packagist?

**Pull request merged**